source/sass/views/projects.scss

```diff
@@ -1,15 +0,0 @@
-.project-image {
-  max-height: 11em;
-}
-
-.projects-row {
-  border-bottom: 1px solid $gray-05;
-}
-
-#view-projects .btn {
-  white-space: initial;
-}
-
-a.pulse-project {
```

Comment:
We no longer have a [Project page/view](https://github.com/mozilla/foundation.mozilla.org/issues/90#issuecomment-289945507) however these rules are still being used on pulse project list so I've
relocated this chunk to `source/js/components/pulse-project-list/pulse-project-list.scss`